tests/unit/test_control_plot_data.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+import numpy as np
+from pandas import DataFrame
```

Comment:
It is customary to `import pandas as pd`

Reply:
Really? I would have thought it has become  a habit, born out of command line usage. Everybody knows what np and pd stand for, but I personally don't like it in library code.

Reply:
> Really? I would have thought it has become a habit, born out of command line usage. Everybody knows what np and pd stand for, but I personally don't like it in library code.

Yes, that´s probably where it came from: https://pandas.pydata.org/docs/getting_started/10min.html#min